### PR TITLE
Updated requirements. Deletes all variants of a certain case and variant...

### PR DESCRIPTION
**Pull request by moonso**
... type before new variants are added

### CODE REVIEW

File: scout/models/variant.py [hunk 1]
```diff
@@ -442,6 +436,7 @@ def end_position(self):
 
     return self.position + (bases - 1)
 
+  # This is exactly the same as variant_id...
```
Comment (robinandeer) [inline scout/models/variant.py]: you can remove this!

Comment (robinandeer) [inline scout/models/variant.py]: Change this across the server/templates to use "variant_id"

**Comment by robinandeer** [comment]
I'm not sure which field you wanted me to use instead of "id_string" but it seems "display_name" is almost the same so I use that! Correct?

**Comment by moonso** [comment]
No it should be ```variant_id```

**Comment by robinandeer** [comment]
Hmmm.. variant id is an MD5 sum though, no?

I'm using ``id_string`` in the sanger email to five the variant an id
https://github.com/Clinical-Genomics/scout/pull/92/files#diff-6415cc6a9574f468510a1ada00d9e49cR762

**Comment by moonso** [comment]
Ok you are probebly right:)

**Comment by robinandeer** [comment]
:+1:

**Pull request merged**